src/helpers/queryDataParser.js

```diff
@@ -0,0 +1,150 @@
+import _filter from 'lodash/filter';
+import { texts } from '../config';
+
+import { QUERY_TYPES } from '../queries';
+import { eventDate } from './dateTimeHelper';
+import { mainImageOfMediaContents } from './imageHelper';
+import { momentFormat } from './momentHelper';
+import { shareMessage } from './shareHelper';
+import { subtitle } from './textHelper';
+
+export const parseEventRecords = (data, skipLastDivider = false) => {
+  return data.map((eventRecord, index) => ({
```

Comment:
missing `?` for `data` like in the other methodes. i will add it, as i am in the file right now.

- [ ] data?